fleet/store/templates.go

```
@@ -0,0 +1,61 @@
+type Template struct {
+	ID         int64
+	Name       string
+	Sources    []string
+	PolicyJSON json.RawMessage
+	CreatedAt  time.Time
+}
+
+const templateCols = `id,name,sources,policy_json,created_at`
+
+func (s *Store) CreateTemplate(ctx context.Context, t *Template) (int64, error) {
+	src, _ := json.Marshal(t.Sources)
```

golangci-lint (2.13.2)

[error] Error return value of `encoding/json.Marshal` is not checked

(errchkjson)